examples/ResponsiveExample.js

```diff
@@ -56,14 +74,4 @@ class ResponsiveExample extends React.Component {
   }
 }
 
-// See react-dimensions for the best way to configure
-// https://github.com/digidem/react-dimensions
-export default Dimensions({
```

Comment:
Can you explain why this was removed?

Reply:
react-dimensions is not maintained now

Reply:
react-dimensions is not maintained now.
https://github.com/digidem/react-dimensions